dsl/procedures/importMicroservices/steps/importMicroservices.groovy

```diff
@@ -45,22 +45,28 @@ if (environmentProjectName && environmentName && clusterName) {
 // write out the docker compose to the workspace directory
 String dir = System.getenv('COMMANDER_WORKSPACE')
 File composeFile = new File(dir, 'docker-compose.yml')
-composeFile << dockerComposeContent
+String composeContent = dockerComposeContent
+composeFile << composeContent.replace('#','').replace("\t", "")
+
 
 def composeConfig
 try {
     composeConfig = DockerClient.readCompose(composeFile)
+    if(composeConfig?.version == '3' || composeConfig?.version == '2') {
```

Comment:
Why is it 3.5, while docker-client documentation states that it should support version 3 (I guess it is 3+)? Have you found any evidences against it? Also, why not numeric comparison?